sim/shaman/totems.go

```diff
@@ -13,6 +13,7 @@ import (
 func (shaman *Shaman) NewAirTotem(sim *core.Simulation) *core.SimpleCast {
 	baseManaCost := 320.0
 	manaCost := baseManaCost * (1 - float64(shaman.Talents.TotemicFocus)*0.05)
+	manaCost -= baseManaCost * float64(shaman.Talents.MentalQuickness) * 0.02
```

Comment:
Add a todo to confirm this is how the calculation works, unless you already have.

Reply:
Mental Quickness reduces instant cast spells by 2% per level, and I wowhead shows the talent impacting totem costs.

Reply:
I meant to check whether the cost reduction is based on the base mana cost, or if its something else. Would be good to check with the enhancement shaman what the actual mana cost is for their totems and compare with this calculation.